flow-typed/types/log-level.js

```diff
@@ -2,3 +2,9 @@
 declare type PKLogLevelObject = { value: number, name: string };
 declare type PKLogLevels = { [level: string]: PKLogLevelObject };
 declare type PKLogLevelTypes = { [level: string]: string };
+declare type LogHandlerType = (messages: any[], context: Object) => void;
+declare type PKLogConfigObject = {
+  level:string,
+  playerVersion:boolean,
```

Comment:
`playerVersion` is used in kaltura player only so move it there (declare it in kaltura player and use the playkit as base type and add `playerVersion`. see https://github.com/kaltura/playkit-js-providers/blob/master/flow-typed/types/media-info.js for example)

Reply:
fixed